db/command.go

```diff
@@ -249,3 +251,39 @@ func ApplyFlags(opts *mopt.FindOneOptions, flags int) {
 		opts.SetOplogReplay(true)
 	}
 }
+
+// RunApplyOpsCreateIndex will create index using applyOps.
+// For versions that support collection UUIDs (<3.6) it uses an insert to system indexes.
+// Later versions use the createIndexes command.
+func (sp *SessionProvider) RunApplyOpsCreateIndex(C, DB string, index bson.D, UUID primitive.Binary, supportsUUID bool, result *interface{}) error {
+	var op Oplog
+
+	// If index version was removed, add it back in
```

Comment:
Could you please expand on this comment?  Why would it not exist?  Etc.

Reply:
It may make more sense to pull this into mongorestore before RunApplyOpsCreateIndex is created, since adding the version back is mongorestore-specific.

Reply:
Sorry -- I meant could you please expand the comment in the code itself.
Looking at mongomirror [FixOutgoingIndexSpec](https://github.com/10gen/mongomirror/blob/6a66616c9508cd57fcea2e7ffa59b1c3dd378867/mongomirror/command_retryer.go#L314-L319), it says that version didn't exist in oplog entries before 3.4.

Reply:
Hm in that case maybe it's best to keep it in RunApplyOpsCreateIndex as a safety measure.